config.json.ctmpl

```diff
@@ -7,6 +7,7 @@
     "bioDataCatalystStagingHost": {{if eq $runContext "fiab"}}"wb-mock-drs-dev.storage.googleapis.com"{{else}}"staging.gen3.biodatacatalyst.nhlbi.nih.gov"{{end}},
     "bioDataCatalystLegacyHost": {{if eq $runContext "fiab"}}"wb-mock-drs-dev.storage.googleapis.com"{{else if eq $environment "prod"}}"gen3.biodatacatalyst.nhlbi.nih.gov"{{else}}"staging.gen3.biodatacatalyst.nhlbi.nih.gov"{{end}},
     "theAnvilHost": {{if eq $runContext "fiab"}}"wb-mock-drs-dev.storage.googleapis.com"{{else if eq $environment "prod"}}"gen3.theanvil.io"{{else}}"staging.theanvil.io"{{end}},
+    "TerraDataRepoHost": {{if eq $runContext "fiab"}}"TBD"{{else if eq $environment "prod"}}"data.terra.bio"{{else}}"staging.data.terra.bio"{{end}},
```

Comment:
We weren't sure what the TDR fiab url should be - I'll update this PR once I find out but wanted to get this up sooner rather than later.

Reply:
try making it the same as what's there for anvil (and all the other fiab resolvers) `"wb-mock-drs-dev.storage.googleapis.com"`

Reply:
oh yeah they sure are all the same aren't they

Reply:
updated to be consistent